internal/pkg/log/service_test.go

```diff
@@ -0,0 +1,72 @@
+package log
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestServiceLogger_VerboseFalse(t *testing.T) {
+	t.Parallel()
+
+	var out strings.Builder
```

Comment:
Tu je upraveny `TestApiLogger_VerboseFalse` test, nemame use-case na zmenu prefixu, iba na pridavanie, a to sa aj testuje.